src/modules/services_manager_service.rb

```diff
@@ -409,6 +415,13 @@ def enable_or_disable(services)
       end
     end
 
+    # Refresh the services information
+    #
+    # This is vitally important during 1st stage, where services were read really early.
```

Comment:
Why is it important to refresh services?

Reply:
Because the information we read during early 1st stage is not reliable (we are reading that info from the instsys). Actually, [this line](https://github.com/yast/yast-yast2/blob/SLE-15-GA/library/systemd/src/lib/yast2/system_service.rb#L421) will prevent several changes to happen.

I will extend the comment.